oslo_vmware/rw_handles.py

```diff
@@ -706,3 +707,28 @@ def close(self):
 
     def __str__(self):
         return "Image read handle"
+
+
+class ImageSwiftUrl(object):
```

Comment:
I am not to sure about adding an "Url" to the module `rw_handles`. I think, it would be nicer, if it would be a `SwiftImageReadHandle`, which can actually read from the swift-url traditionally, but also with some extra methods needed to get the required parameters to write the image.

But I understand that increases the scope. Still we could start with just leaving those functions unimplemented

Reply:
I just think every "FileReadHandle" should be able to support that interface.

Reply:
Ah, but it would require some major(?) changes of the file-handles not to open the connection immediately.

Reply:
After some more reading, that seems indeed a rather large change. So, maybe we postpone it for a second round